jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -153,7 +153,7 @@ def _return_spectrum_with_correct_units(flux, wcs, metadata, data_type, target_w
             for cunit_num in (3, 1):
                 cunit_key = f"CUNIT{cunit_num}"
                 ctype_key = f"CTYPE{cunit_num}"
-                if cunit_key in hdr and 'WAVE' in hdr[ctype_key]:
+                if cunit_key in hdr and 'WAV' in hdr[ctype_key]:
```

Comment:
is there any risk here of this catching something we don't want it to?

Reply:
looks like there are (currently) three possible matches (WAVE, WAVN, AWAV).  Do we need any order of preference to these or are there no cases where multiple ones would be present?

Reply:
> Do we need any order of preference to these or are there no cases where multiple ones would be present?

I don't think any sane cube WCS has more than 1 spectral axis at a given time?

If it does, how do we even know which one to use?

Reply:
right now it would just be somewhat random.... but no need to make the logic more complicated if that case doesn't exist.  Let's leave as-is for now.  Thanks!